distribution/docker/Dockerfile

```diff
@@ -51,12 +52,16 @@ RUN --mount=type=cache,target=/root/.m2 VERSION=$(mvn -B -q org.apache.maven.plu
 
 FROM busybox:1.34.1-glibc as busybox
 
-FROM gcr.io/distroless/java$JDK_VERSION-debian11
+FROM $BASE_IMAGE
+
 LABEL maintainer="Apache Druid Developers <dev@druid.apache.org>"
 
 COPY --from=busybox /bin/busybox /busybox/busybox
-RUN ["/busybox/busybox", "--install", "/bin"]
-
+RUN ["/busybox/busybox", "sh", "-c", "if [ ! -x \"$(command -v bash)\" ]; then \
+                                        exec /busybox/busybox --install /bin; \
```

Comment:
nit, do you know if `exec` is required here?

Reply:
Right, this is not required :)